### PR TITLE
Use computed columns for merkle tree roots

### PR DESCRIPTION
**Pull request by imabdulbasit**
Closes #<ISSUE_NUMBER>

### This PR:

uses computed columns for merkle tree roots

https://github.com/EspressoSystems/espresso-sequencer/pull/1451

### CODE REVIEW

File: migrations/V20__merkle_root_columns.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ALTER TABLE header
```
Comment (jbearer) [inline migrations/V20__merkle_root_columns.sql:1]: This migration should go in the sequencer

File: src/data_source/storage/sql.rs [hunk 2]
```diff
@@ -3183,6 +3183,11 @@ pub mod testing {
                 value BYTEA  NOT NULL UNIQUE
             );
         
+
+            ALTER TABLE header
```
Comment (jbearer) [inline src/data_source/storage/sql.rs:3187]: Could just include this in the `CREATE TABLE` statement

Comment (imabdulbasit) [reply src/data_source/storage/sql.rs:3187]: dont we need a separate migration for this?

Comment (imabdulbasit) [reply src/data_source/storage/sql.rs:3187]: this is for test merkle tree so didn't want to add it in migration

**Review by jbearer — APPROVED** [review]

**Pull request merged**